gallery.js

```diff
@@ -2,12 +2,19 @@ let fs = require('fs');
 
 let imageFileNames = fs.readdirSync('site/images');
 
-// This code is just here to demonstrate how fs.readdirSync works
-
-console.log('sites/images contains the following files:')
+//This code is just here to demonstrate how fs.readdirSync works
 
+console.log('sites/images contains the following files:');
+let imageGallery;
 for (let i = 0; i < imageFileNames.length; i++) {
   let fileName = imageFileNames[i];
-
-  console.log(`${i + 1}. ${fileName}`);
+  if (i === 0) {
+    imageGallery = `<img src="images/${fileName}" height="400" width="550">`;
```

Comment:
You want to avoid attributes like `height` and `width` in HTML. Use CSS to set things like that.

Reply:
It can be useful to create little helper functions to generate HTML, like:

```javascript
function imgTag(src) {
  return `<img src="${src}">`;
}
```

Then elsewhere:

```javascript
function getGalleryHTML(fileNames) {
  let html = '';

  for (let fileName in fileNames) {
    html += imgTag(fileName);
  }

  return html;
}

function getTemplate(content) {
  return `
<!DOCTYPE html>
<html>
  <head>
    <title>Gallery</title>
  </head>
  <body>
    ${content}
  </body>
</html>
`;
}
```

You can use template literals across multiple lines like that.